src/bo4e/validators.py

```diff
@@ -18,7 +19,46 @@ def check_list_length_at_least_one(instance, attribute, value):
 def einheit_only_for_abschlagstyp_absolut(instance, attribute, value):
     """
     Check that einheit is only there if abschlagstyp is absolut.
-    Currently (2021-12-15) only used in COM AufAbschlag.
+    Currently, (2021-12-15) only used in COM AufAbschlag.
     """
     if value and (not instance.auf_abschlagstyp or (instance.auf_abschlagstyp != AufAbschlagstyp.ABSOLUT)):
         raise ValueError("Only state einheit if auf_abschlagstyp is absolute.")
+
+
+# pylint: disable=unused-argument
+def check_list_length_is_one_or_two(instance, attribute, value):
+    """
+    Check if list length is one or two.
+    So far only used in StandorteigenschaftenGas.
+    """
+    if len(instance.netzkontonummern) == 0:
+        raise ValueError("Netzkontonummern must not be empty.")
+    if len(instance.netzkontonummern) > 2:
+        raise ValueError("Maximum number of Netzkontonummern is 2.")
+
```

Comment:
Man kann mit {attribute.name} den Value Error verallgemeinern.

Reply:
Löst aber nicht das problem, dass zeile 36 und 34 noch auf die netzkontonummern verweisen. Stattdessen müsste man (analog zu `get_inclusive_start()...)` die liste hinter einem Protocol verstecken, das von der konkreten Klasse implementiert wird. 
Wenn man das ganz allgemein machen wollte, dann müsste eine klasse, die den validator nutzen darf ungefähr sowas wie ein Dictionary zurückgeben können, dass die in ihr vorhandenen Listen auf ein (min,max)-length tuple mapped. Aber die extra-komplexität die wir dadurch einhandeln lohnt imo nicht.

Reply:
Kann man es nicht analog zu `check_list_length_at_least_one` über den value machen oder übersehe ich etwas?